docs/querying/math-expr.md

```diff
@@ -43,7 +43,7 @@ This expression language supports the following operators (listed in decreasing
 |^|Binary power op|
 |*, /, %|Binary multiplicative|
 |+, -|Binary additive|
-|<, <=, >, >=, ==, !=|Binary Comparison|
+|&lt;, &le;, &gt;, &ge;, ==, &ne;|Binary Comparison|
```

Comment:
The `!=` shows up as `≠`, not sure if that's intentional

Reply:
hmm, maybe hte way to fix this is to just wrap all the symbols in code ticks

Reply:
```suggestion
|&lt;, &le;, &gt;, &ge;, ==, !=|Binary Comparison|
```